arangod/RocksDBEngine/RocksDBSha256Checksum.h

```diff
@@ -0,0 +1,91 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2021 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Steemann
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "ApplicationFeatures/ApplicationServer.h"
+#include "Basics/Mutex.h"
+
+#include <openssl/evp.h>
+#include <rocksdb/file_checksum.h>
+#include <rocksdb/listener.h>
+
+#include "Logger/LogMacros.h"
+
+
```

Comment:
No logging used here. This include should go into the .cpp file.
```suggestion
#include "Logger/LogMacros.h"
```

Reply:
Addressed